test/consumer/Dockerfile

```
@@ -8,7 +8,7 @@ COPY . .
 RUN CGO_ENABLED=0 GOOS=linux go build -ldflags="-s -w" -o consumer main.go  
   
 FROM alpine:latest  
-RUN apk --no-cache add ca-certificates  
+RUN apk --no-cache add ca-certificates
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)